pkg/faketikv/node.go

```diff
@@ -218,3 +218,13 @@ func (n *Node) Stop() {
 	n.wg.Wait()
 	simutil.Logger.Infof("node %d stopped", n.Id)
 }
+
+func (n *Node) decStoreSize(size uint64) {
```

Comment:
Could you make the name more specific? I can't tell from the name whether it decreases the available size or the used size.

Reply:
`decStoreSize` here means that decreasing the store size. The available size of the store will increase as well as the used sized will be decreased. And `incStoreSize` is similar.

Reply:
em, quote "available size of the store will increase as well as the used sized will be decreased", but the code actually decreases availableSize and increases usedSize. That makes me even more confused...

Reply:
edited